### ISSUE TITLE
ResNet101 weights it seems?

### ISSUE THREAD

**Issue by sayakpaul** [facebookresearch/vissl#482]
## 📚 VISSL Documentation

Documentation: https://vissl.ai/tutorials/Using_a_pretrained_model_for_inference_V0_1_6. 

I am referring to this section (Download the ResNet-50 Simclr weights from the Model Zoo) of the above-mentioned tutorial. Even though it explicitly mentions ResNet50 the download link seems to be indicating the weights of ResNet101:

```
wget -q -O /content/resnet_simclr.torch https://dl.fbaipublicfiles.com/vissl/model_zoo/simclr_rn101_1000ep_simclr_8node_resnet_16_07_20.35063cea/model_final
```

Possible to ensure the validity i.e., which model architecture do the weights correspond to?

**Comment by iseessel** [comment]
Indeed we should just change ResNet50 to ResNet101 in the tutorail -- nice catch!

**Comment by sayakpaul** [comment]
Thanks! So if we change the identifier to rn50, should it be ResNet50? Or is there a different handle?

**Comment by sayakpaul** [comment]
@iseessel just a gentle ping.

**Comment by iseessel** [comment]
Sorry about the delay -- thanks for pinging me, missed this! 

There are two options: 

1. Download the Resnet-50 weights instead of the Resnet-101 weights. You can see these here: https://github.com/facebookresearch/vissl/blob/main/MODEL_ZOO.md#SimCLR

2. Continue to use the Resnet-101 weights and change the config to use Resnet-101 model here: https://github.com/facebookresearch/vissl/blob/main/configs/config/pretrain/simclr/models/resnext101.yaml

**Comment by sayakpaul** [comment]
Thanks much. Do you want me to keep the issue open until the tutorial is updated? I can also do a PR with the correction.

**Comment by iseessel** [comment]
Yeah if you don't mind sending a correction PR that would be great!

Let's do option 2 to give the tutorials some variety in terms of types of models used.

**Comment by sayakpaul** [comment]
Alright. Will do it.
